### DISCUSSION TITLE
Accuracy of gmx_MMPBSA when grabbing interval frames from a GROMACS simulation

### DISCUSSION THREAD

**Discussion by alanjchang** [Valdes-Tresanco-MS/gmx_MMPBSA#538] [Q&A]
Is gmx_MMPBSA more accurate when interval = 1 in the input file, meaning that it analyzes every single of the trajectory, or is it more accurate when interval = 10 for example, so it analyzes every 10 frames. In other words, does the accuracy increase/decrease if gmx_MMPBSA only grabs every n frames vs if it grabs every frame?

**Comment by Valdes-Tresanco-MS** [comment]
That depends on the stability of your trajectory. The calculation is deterministic, so you will get the same energy values ​​for the same frames regardless of the interval. But if your trajectory is not stable, this can affect the average. Technically, the interval simply reduces the number of frames to reduce the computation time.
